readthedocs/projects/tasks.py

```diff
@@ -902,63 +907,62 @@ def move_files(version_pk, hostname, html=False, localmedia=False,
     if html:
         from_path = version.project.artifact_path(
             version=version.slug,
-            type_=version.project.documentation_type,
+            type_=doctype,
         )
         target = version.project.rtd_build_path(version.slug)
         Syncer.copy(from_path, target, host=hostname)
 
-    if 'sphinx' in version.project.documentation_type:
-        if search:
-            from_path = version.project.artifact_path(
-                version=version.slug,
-                type_='sphinx_search',
-            )
-            to_path = version.project.get_production_media_path(
-                type_='json',
-                version_slug=version.slug,
-                include_file=False,
-            )
-            Syncer.copy(from_path, to_path, host=hostname)
+    if search:
```

Comment:
Is this not going to try and run this logic on mkdocs builds?

Reply:
Not, search is false when the doctype is `mkdocs`

https://github.com/stsewd/readthedocs.org/blob/517dd2d887fc5322cb3dee79431d6b23de9aa15c/readthedocs/projects/tasks.py#L813-L818